src/styles/main.scss

```diff
@@ -1,3 +1,112 @@
-body {
+@import "./variables";
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+.month {
+  position: relative;
+  width: $mounth-width;
+  padding: 10px;
+  margin: 100px auto 0;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.